src/agent/centralized_configuration/src/centralized_configuration.cpp

```diff
@@ -6,18 +6,23 @@
 
 namespace centralized_configuration
 {
-    boost::asio::awaitable<module_command::CommandExecutionResult>
-    CentralizedConfiguration::ExecuteCommand(const std::string command) // NOLINT(performance-unnecessary-value-param)
+    boost::asio::awaitable<module_command::CommandExecutionResult> CentralizedConfiguration::ExecuteCommand(
+        const std::string command,       // NOLINT(performance-unnecessary-value-param)
```

Comment:
Why are these `NOLINT` for?

Reply:
Due to a performance warning related to parameter passing by value. Previously it was for the command parameter and now it was added also for “parameters”.